pkg/test/kube/dump.go

```diff
@@ -36,6 +37,14 @@ import (
 	"istio.io/istio/pkg/test/scopes"
 )
 
+const (
+	maxCoreDumpedPods = 5
+)
+
+var (
+	coreDumpedPods = atomic.NewInt32(0)
```

Comment:
I think this is fine but fwiw this will be a per-package limit I think, since each package is its own executable

Reply:
> max 5 * suites is still only like 30

I may optimize later, didn't want to add complexity such as storing this in a file